pkg/kubelet/cm/cpumanager/cpu_manager.go

```diff
@@ -450,7 +475,7 @@ func (m *manager) reconcileState() (success []reconciledContainer, failure []rec
 				// was allocated.
 				_, _, err := m.containerMap.GetContainerRef(containerID)
 				if err == nil {
-					klog.V(4).InfoS("ReconcileState: ignoring terminated container", "pod", klog.KObj(pod), "containerID", containerID)
+					lh.V(4).Info("ignoring terminated container", "containerID", containerID)
```

Comment:
Why not update the `logger` with `containerID` and use that for logging below?

Reply:
yep, you have it in the next comment